app/src/hooks.client.ts

```diff
@@ -1,25 +1,25 @@
-import { handleErrorWithSentry } from '@sentry/sveltekit';
+import { captureException } from '@sentry/sveltekit';
 import { error as logErrorToFile } from 'tauri-plugin-log-api';
-import type { NavigationEvent } from '@sveltejs/kit';
+import type { HandleClientError } from '@sveltejs/kit';
 
-function myErrorHandler({ error, event }: { error: any; event: NavigationEvent }) {
-	console.error(error.message + '\n' + error.stack);
-	logErrorToFile(error.toString());
-	console.error('An error occurred on the client side:', error, event);
-}
+// eslint-disable-next-line func-style
+export const handleError: HandleClientError = ({ error, status }) => {
```

Comment:
Change this to `export function handleError({ error: unknown, status: number }): HandleClientError`?

Reply:
Yeah that works too, unfortunately we can't use the full function type declaration (incl. parameters) with the function declaration style, which is why I went this route ^^.

But `error` and `status` here are pretty straight forward and the return can just be `ReturnType<HandleClientError>` :+1: